infra/ansible/roles/nginx_config_helper/files/update-nginx-config.py

```
@@ -44,7 +67,45 @@ def write_normalized(path: Path, text: str) -> bool:
+            depth += 1
+        elif text[index] == "}":
+            depth -= 1
+            if depth == 0:
+                return index + 1
+    return None
+
+
+def listens_on_443(server_block: str) -> bool:
+    for listen_match in LISTEN_DIRECTIVE_PATTERN.finditer(server_block):
+        for token in listen_match.group("args").split():
+            if token == "443" or token.endswith(":443"):
```

Ruff (0.15.11)

[error] Possible hardcoded password assigned to: "token"

(S105)